### DISCUSSION TITLE
Make screenshot plugin to standard funcionality?

### DISCUSSION THREAD

**Discussion by Dreibuchstaben** [EDCD/EDMarketConnector#1470] [General]
Hi,

the current version of the screenshot plugin stopped working with 5.3.0.

I always enjoyed that funcionality, because it is super handy and helpful.

So I thought, that should be a must have of the EDMC. What do you think?

**Comment by Athanasius** [comment]
For the benefit of anyone else finding this, I see you also opened an issue with the third-party developer of that plugin: https://github.com/NoFoolLikeOne/EDMC-Screenshot/issues/51

And, no, we won't generally add more functionality to the core of EDMC, it's already a lot of work to maintain and improve as it is. This is *precisely* what the plugin functionality is for.
